src/cljs/org/broadinstitute/firecloud_ui/page/workspace/method_config_importer.cljs

```diff
@@ -67,16 +67,34 @@
                    (str "Selected Method Configuration Namespace : " (:selected-mc-namespace @state))
                    [:hr]
                    "Destination Name : "
-                   (style/create-text-field {:style {:ref "destinationName"}})
+                   (style/create-text-field {:id "dest_name_id" :style {:ref "destinationName"}})
                    [:br]
                    "Destination Namespace : "
-                   (style/create-text-field {:style {:ref "destinationNamespace"}})
+                   (style/create-text-field {:id "dest_namespace_id" :style {:ref "destinationNamespace"}})
                    [:br]
+                   [comps/Button  {:title_text "Pre-populate names"
+                                   :icon :gear
+                                   :onClick (fn []
+                                              (let [
```

Comment:
Put the first item of the let on the same line, and also don't put the closing ] on its own line.  I see that the line is getting long--to alleviate this, you can bring the onClick function down to the next line, saving 9 characters, or bring the properties for the button (declared on line 75) down to the next line like the other two buttons.